requirements/prod.in

```diff
@@ -21,6 +21,7 @@ gunicorn==20.0.4
 hiredis==2.2.3
 ipaddress==1.0.23
 lxml==4.9.3
+markus==4.2.0
```

Comment:
I think you want to install the datadog bits, too:

https://github.com/mozilla-services/socorro/blob/ab7e11e92e9ad15e6971ebe6a490320d7eb7411b/requirements.in#L33

Using the datadog bits gives you tags and histograms which regular stats doesn't have, but influxdb supports.